.changeset/famous-icons-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': major
```

Comment:
This should be either a `minor` or target the `next` branch (for v5)

Reply:
oh sorry I just had that option by using `yarn changeset` (first time using it), now it should be updated